academy/ibc/channels.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "IBC/TAO"
+order: 
+description: 
+tag: deep-dive
+---
+
+## Channels
+
+Connections and clients comprise the main components of the transport layer in IBC. However, application to application communication in IBC is conducted over **channels**, which route between an application module such as the module which handles ICS20 token transfers on one chain, and the corresponding application module on another one. These applications are namespaced by **port identifiers** such as 'transfer' for ICS20 token transfers, or 'interchainaccounts' for the Interchain Accounts module. 
```

Comment:
the ICA port identifiers are not 'interchainaccounts' right? but:
{
 // Controller Chain
 SourcePortId: `icacontroller-<owner-account-address>`,
 SourceChannelId: `<channel-id>`,
 // Host Chain
 CounterpartyPortId: `icahost`,
 CounterpartyChannelId: `<channel-id>`,
}

Reply:
good catch 👍